civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
     }
   }
 }
+
+/*
+* This is a TEMP fix to fit the text of input controls like date picker within input box
+* It is hardcoded to 13px for now.
+*/
+.chr_leave-request-modal__input-font-size {
+  font-size: 13px;
+}
+
+.chr_leave-request-modal__span-expiry {
```

Comment:
This looks an ad-hoc fix, did you see what the reason is for the misalingment in the first place